src/panel/category/CategoryPanel.jsx

```diff
@@ -165,6 +165,7 @@ const CategoryPanel = ({ poiFilters = {}, bbox }) => {
       floatingItemsLeft={[
         isMobile && shouldShowBackToQwant() && <BackToQwantButton key="back-to-qwant" isMobile />,
       ]}
+      fitContent={!pois || pois.length === 0 ? 'default' : 'minimized'}
```

Comment:
Same thing here, it should be an array.

Moreover, you must not pass `'default'` in the non-error case, as it breaks the normal behavior. You should pass an empty array instead, which tells `Panel` to use static sizes instead for all panel sizes.

Reply:
thanks for the explanations, I hadn't use this mechanism until today and it wasn't super clear for me how to use it properly.
Fixed it, hope it's ok now. It seems to work nice according to my tests :)